components/certificate-mgt/org.wso2.carbon.identity.certificate.management.core/src/main/java/org/wso2/carbon/identity/certificate/management/core/model/Certificate.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.certificate.management.core.model;
+
+/**
+ * This class holds the certificate information.
+ */
+public class Certificate {
+
+    private String id;
+    private String name;
+    private String certificate;
```

Comment:
Indicate in which format the certificate is represented

Reply:
Fixed with https://github.com/wso2/carbon-identity-framework/pull/6043/commits/f33815ecd75c1d6ee0d75bd018a79ef7b6369cc2